.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: PythonCI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ["3.5", "3.8"]
+        tox-version: ["django22", "quality"]
```

Comment:
this configuration would run quality with both python versions, we should limit it to python 3.8

Reply:
Excluded `quality` tests for `python3.5`.